### PR TITLE
Shania resources styling

### PR DESCRIPTION
**Pull request by Ulam22**
## Results ##
preview1
preview2

ahh I don't know how to change the individual background colors for the cards. I tried doing the whole nth-child thing but that didn't work.

Also, I need to change the spacing between the cards when in mobile view because it looks like this
card-mobile

### CODE REVIEW

File: src/styles/resources.scss [hunk 1]
```diff
@@ -4,10 +4,31 @@
...
+    color: white;
+    line-height: 33px;
   }
 
+
   &__rounded-card {
     border-radius: 18px;
     cursor: pointer;
+    background-color: #386A3F;
+    min-height: 100%;
```
Comment (nupur-n) [inline src/styles/resources.scss:18]: could you try giving the rounded-card a top and bottom margin? Wondering if that would help with the spacing on mobile

File: src/data/resources.js [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
-    link: "http://google.com",
-    imageURL:
-      "https://upload.wikimedia.org/wikipedia/commons/9/90/Blank_flag_large.PNG",
-  },
-  {
-    name: "PROJECT SUBMISSION RULES",
-    link: "http://google.com",
-    imageURL:
-      "https://upload.wikimedia.org/wikipedia/commons/9/90/Blank_flag_large.PNG",
+    name: "Camp Gear",
```
Comment (nupur-n) [inline src/data/resources.js:7]: could you add a "color" field here that tells you what color to render the card?

**Review by snwessel — APPROVED** [review]
Beautifuuuul!

**Comment by snwessel** [comment]
image
Adding a photo of what it looks like now for other reviewers (colors may be slightly different)

File: src/styles/resources.scss [hunk 3]
```diff
@@ -4,10 +4,35 @@
...
     cursor: pointer;
+    min-height: 100%;
+    margin-bottom: 15px;
+    
+    .card {
+      margin-left: 10px;
+      margin-bottom: 10px;
+    }
+  img {
+width: 200px;
```
Comment (feliciazhang) [inline src/styles/resources.scss:26]: weird fomatting here but gonna approve anyway

**Review by feliciazhang — APPROVED** [review]

**Pull request merged**